targets/mpc85xx-p1010

```diff
@@ -7,4 +7,6 @@ device('sophos-red-15w-rev.1', 'sophos_red-15w-rev1', {
 
 -- TP-Link
 
-device('tp-link-tl-wdr4900-v1', 'tplink_tl-wdr4900-v1')
+device('tp-link-tl-wdr4900-v1', 'tplink_tl-wdr4900-v1', {
+	broken = true,	-- Kernel exceeds max-size
```

Comment:
Broken devices must at least build. As this is not the case for the WDR4900, the entry should be commented out rather than marked as broken.